wheels/view/errors.cfm

```diff
@@ -15,14 +15,15 @@ public string function errorMessagesFor(
 	required string objectName,
 	string class,
 	boolean showDuplicates,
-	boolean encode
+	boolean encode,
+	boolean includeAssociations
```

Comment:
Should this have a default of "false" - and for that matter, shouldn't the other arguments have defaults?

Reply:
yes it should haha, I kind of didn't include the default because the other args don't have defaults haha